src/renderer/dx/DxRenderer.cpp

```diff
@@ -250,6 +549,8 @@ DxEngine::~DxEngine()
             default:
                 THROW_HR(E_NOTIMPL);
             }
+
+            RETURN_IF_FAILED(_SetupTerminalEffects());

```

Comment:
Should effects be non-critical?

Reply:
I agree, this should probably just be 
```suggestion
            LOG_IF_FAILED(_SetupTerminalEffects());
```
And we should probably file a follow-up task to propagate some sort of dialog up to the app to say _"We couldn't initialize the graphics effects for whatever reason."_